plugins/inputs/cosmic/README.md

```diff
@@ -0,0 +1,115 @@
+# Cosmic Input Plugin
+
+This plugin will collect statistics about virtualmachines, volumes and public ip adresses available in Cosmic.
+It uses [Cosmic API](https://apidoc.cosmiccloud.io/) to query the data.
+
+See [Cosmic](https://github.com/MissionCriticalCloud/cosmic/) for more information on the infrastructure orchestrator.
+
```

Comment:
This plugin looks it probably has the potential to create lots of series, similar to many of container orchestration plugins that use UUIDs.  If there are any opportunities to remove UUIDs in favor of names, without it causing problems due to lack of uniqueness, it might allow users to be able to store data for longer.  This is especially useful if the UUID changes frequently, and less so on things that rarely change like user UUIDs.

Ultimately there is probably limited changes you can make, so perhaps we should add the "Series Cardinality Warning" like the [DC/OS](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/dcos#dcos-input-plugin) has?